.github/workflows/release-files.yml

```diff
@@ -0,0 +1,137 @@
+name: hdf4 dev release-files
+
+# Controls when the action will run. Triggers the workflow on a schedule
+on:
+  workflow_call:
+    inputs:
+      use_tag:
+        description: 'Release version tag'
+        type: string
+        required: false
+        default: snapshot
+      use_environ:
+        description: 'Environment to locate files'
+        type: choice
+        required: true
+        default: snapshots
+        options:
+        - snapshots
+        - release
+      file_base:
+        description: "The common base name of the source tarballs"
+        required: true
+        type: string
+      file_branch:
+        description: "The branch name for the source tarballs"
+        required: true
+        type: string
+      file_sha:
+        description: "The sha for the source tarballs"
+        required: true
+        type: string
+
+# Minimal permissions to be inherited by any job that doesn't declare its own permissions
+permissions:
+  contents: read
+
+# Previous workflows must pass to get here so tag the commit that created the files
+jobs:
+  create-tag:
+    runs-on: ubuntu-latest
+    permissions:
+        contents: write # In order to allow tag creation
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - name: Get Sources
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - run: |
+          git checkout ${{ inputs.file_sha }}
+
+      - uses: rickstaa/action-create-tag@v1
+        id: "tag_create"
+        with:
+          commit_sha: ${{ inputs.file_sha }}
+          tag: "${{ inputs.use_tag }}"
```

Comment:
inputs.use_tag means this will ask [a user] to input the release tag?
current formats for release tags:
    hdf-4_vminor_vrelease[-vpatch] for HDF4
    hdf5-1_vminor_vrelease[-vpatch]for HDF5
tag should be applied when release is official and certain, especially so if the tag is public as soon as it is created.  Similarly, release files should only be publicly available when release is final.  Any change after files are put in public release location will require a patch release.

Reply:
Right - the use of inputs prevents the script from running automatically.